src/fairseq2/models/wav2vec2/model.py

```diff
@@ -42,6 +42,8 @@ class Wav2Vec2Model(Model):
     final_target_proj: Linear
     num_distractors: int
     logit_temp: float
+    diversity_loss_weight: float
```

Comment:
See my comment below. I think loss weights should only be specified as parameters of `compute_loss()`.